Homework5/Vacations/Vacation.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vacations
+{
+    internal class Vacation
+    {
+        public List<Employee> SetOfEmployees { get; set; }
+        public Vacation()
+        {
+            SetOfEmployees = new List<Employee>();
+        }
+
+        public void Add(Employee employee)
+        {
+            SetOfEmployees.Add(employee);
+        }
+
+        public int GetAverageDaysOfVacations()
+        {
+            int days = 0;
+            TimeSpan date = new TimeSpan(0, 0, 0);
+
+            foreach (Employee item in SetOfEmployees)
+            {
+                date = item.LastDay - item.FirstDay;
+                days = days + date.Days;
+            }
+            return days / SetOfEmployees.Count;
```

Comment:
This is a working approach, but why don`t u use Linq 
https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.average?view=net-6.0
https://stackoverflow.com/questions/4590704/get-average-using-linq